### ISSUE TITLE
Calling CreateSession() returns 400 Bad Request

### ISSUE THREAD

**Issue by cs-NET** [opentok/Opentok-.NET-SDK#229]
I debugged via the HelloWorld test app and it appears the following is being sent:

```
location=&p2p.preference=disabled&archiveMode=manual&e2ee=False&archiveName=&archiveResolution=640x480
```

If you remove `&archiveName=&archiveResolution=640x480` the call succeeds, I am guessing these parameters cannot be sent if `archiveMode=manual`?

**Comment by Tr00d** [comment]
Hi @cs-NET,

Sorry for the long delay, this issue wasn't on my radar.
Indeed, you're right. I intend to release a new version with the fix in the following days

**Comment by Tr00d** [comment]
Hi @cs-NET,

A fix has been deployed with version [3.11.2](https://github.com/opentok/Opentok-.NET-SDK/releases/tag/v3.11.2).
